### DISCUSSION TITLE
Collaboration Design Patterns > Affecting others > Assistive Actions

### DISCUSSION THREAD

**Discussion by giscus** [pgpais/live-framework-for-cooperative-games#105] [Framework Discussions]
# Collaboration Design Patterns > Affecting others > Assistive Actions

https://www.lfcooperativegames.com/reports/new

**Comment by hg65we** [comment]
Using a ship's Fog Consumable or a flying squadron's Patrol Fighters Consumable can help allies by hiding them from enemies and providing anti-aircraft defense, respectively
